dotcom-rendering/src/lib/routeUtils.ts

```diff
@@ -0,0 +1,300 @@
+/**
```

Comment:
where a file has been copied from another repo it's helpful to include a "This file was copied from" comment, like this example -
https://github.com/guardian/dotcom-rendering/blob/main/dotcom-rendering/src/components/marketing/lib/articleCountOptOut.ts#L2